index.js

```diff
@@ -37,16 +38,38 @@ program
           options.model,
           Temperature
         );
-        return response.choices[0].message.content;
+
+        return {
+          content: response.choices[0].message.content,
+          tokensInfo: {
+            prompt: response.usage.prompt_tokens,
+            response: response.usage.completion_tokens
+          }
+        };
       }));
       
-      const output = responses.join('\n\n=================================================================================\n\n'); 
+      const output = responses.map(r => r.content).join('\n\n=================================================================================\n\n'); 
       if (options.output) {
         fs.writeFileSync(options.output, output); // Save all responses to the output file
         console.log(`File saved to ${options.output}`);
       } else {
         console.log(output);
       }
+
+      if (options.tokenUsage) {
+        const { totalPromptTokens, totalResponseTokens } = responses.reduce(
+          (acc, r) => {

```

Comment:
Same as the previous comment. I'd prefer more meaningful names than `acc` and `r`

Reply:
Fixed with https://github.com/aamfahim/explainer.js/pull/23/commits/7b0c2f2c471cb50bd15c4f8c1a2011b8bcf9e2e4

Reply:
variable `r` is addressed however, acc still remains the same, I would prefer a more descriptive variable name.

Reply:
variable `r` is addressed however, acc still remains the same, I would prefer a more descriptive variable name.

Reply:
Fixed with https://github.com/aamfahim/explainer.js/pull/23/commits/1df07f116afe4dfaf3b2e5e5b3910b5a7d1bb6f4

Reply:
I forgot to rename the return value. Sorry about that. Fixed with https://github.com/aamfahim/explainer.js/pull/23/commits/bcd6c05ef4aecdfd7a2fd0ee0924d5737660e7d9